mono_repo/mono_pkg.yaml

```diff
@@ -6,7 +6,8 @@ dart:
 stages:
 - smoke_test:
   - group:
-    - dartfmt
+    # skipping while https://github.com/dart-lang/dart_style/issues/799 is live
```

Comment:
personally I would prefer just going with the new style until the next dev release

Reply:
Eh, then we have to ignore this on `stable`.

Reply:
we should only be checking formatting on one sdk